src/System.Management.Automation/FormatAndOutput/common/FormattingObjectsDeserializer.cs

```diff
@@ -586,8 +587,10 @@ internal sealed partial class TableHeaderInfo : ShapeInfo
         internal override void Deserialize(PSObject so, FormatObjectDeserializer deserializer)
         {
             base.Deserialize(so, deserializer);
+            // The "repeatHeader" property was added later (V5, V6) and presents an incompatibility when remoting to older version PowerShell sessions.
```

Comment:
Please add new line before the line 590 - it is CodeFactor report.

Reply:
Will do.